src/Sparrow.Server/ByteString.cs

```diff
@@ -658,7 +670,7 @@ public sealed class ByteStringContext(SharedMultipleUseFlag lowMemoryFlag, int a
         internal static readonly int ExternalAlignedSize;
 
         public const int MinBlockSizeInBytes = 4 * 1024; // If this is changed, we need to change also LogMinBlockSize.
-        public const int MaxAllocationBlockSizeInBytes = 256 * MinBlockSizeInBytes;
+        public const int MaxAllocationBlockSizeInBytes = 4 * 256 * MinBlockSizeInBytes;
```

Comment:
By working on very cheap queries I found that the 1Mb limit for the max allocation block size causes excess block allocation and therefore a lot of extra work in the allocation for almost no gain, because we consistently burn through bigger than 1Mb blocks.